src/StackAdmin/AzureRM/AzureRM.psd1

```diff
@@ -48,16 +48,16 @@ CLRVersion='4.0'
 ProcessorArchitecture = 'None'  
   
 # Modules that must be imported into the global environment prior to importing this module  
-RequiredModules = @(@{ModuleName="AzureRM.Profile"; RequiredVersion="3.4.0"},
-  @{ModuleName="Azure.Storage"; RequiredVersion="1.0.5.4"}  
-  @{ModuleName="AzureRM.Compute"; RequiredVersion="3.2.1"},
-  @{ModuleName="AzureRM.Dns"; RequiredVersion="3.4.0"},
-  @{ModuleName="AzureRM.KeyVault"; RequiredVersion="3.4.0"},
-  @{ModuleName="AzureRM.Network"; RequiredVersion="1.0.5.4"},
-  @{ModuleName="AzureRM.Resources"; RequiredVersion="4.4.0"},
-  @{ModuleName="AzureRM.Storage"; RequiredVersion="1.0.5.4"},
-  @{ModuleName="AzureRM.Tags"; RequiredVersion="3.4.0"},
-  @{ModuleName="AzureRM.UsageAggregates"; RequiredVersion="3.4.0"}) 
+RequiredModules = @(@{ModuleName="AzureRM.Profile"; RequiredVersion="3.4.1"},
```

Comment:
I think we will need to edit this - the profile version should be updated and the other modules they get from the current version, but nothing else.